### ISSUE TITLE
Banning false positive

### ISSUE THREAD

**Issue by AskAlexSharov** [anacrolix/torrent#723]
I have 2 servers on Hetzner. 
First serving files, another downloading. 80 files, total size 200gb. 
It works very well until some point (sometime after 15 min, sometime after 1 hour). 
Then client does ban server - only 1 ip in ban list.

**Comment by anacrolix** [comment]
Can you try with `anacrolix/torrent@smartban`?

**Comment by anacrolix** [comment]
Yeah the smartban branch would have made that a certainty if you got failures :). Either a data issue, or bug in anacrolix/torrent. However the other logging observations you made are still very relevant. Thanks for following up!
